ffs/filcold/filcold.go

```diff
@@ -81,14 +81,14 @@ func (fc *FilCold) Fetch(ctx context.Context, pyCid cid.Cid, piCid *cid.Cid, wad
 	return ffs.FetchInfo{RetrievedMiner: miner, FundsSpent: fundsSpent}, nil
 }
 
-func (fc *FilCold) calculatePieceSize(ctx context.Context, c cid.Cid) (api.DataSize, error) {
+func (fc *FilCold) calculateDealPiece(ctx context.Context, c cid.Cid) (abi.PaddedPieceSize, cid.Cid, error) {
```

Comment:
I preferred to avoid using `api` namespace here since is a Lotus package.
Also, I decided using `abi.PaddedPieceSize` since using `uint64` is ambiguous regarding if the size includes padding or not. It's a technical detail that matters in using the right size type for calculating price. For other minor things, we can still use `uint64`. It's just a type-safety mesaure.

Saying it differently: `abi.PaddedPieceSize` always indicates padded data which is the real size it will take in the miner's sector.